apps/e2e-tests/src/process-compose/e2e.spec.ts

```diff
@@ -240,7 +233,6 @@ describe.sequential('E2E Tests with process-compose', () => {
       // Make sure that the feeds info is updated
       for (const [id, data] of entriesOf(currentFeedsInfo)) {
         const { round, value } = data;
```

Comment:
Maybe it will help if we rename some of the vars. I think it might improve readability
  - `currentFeedsInfo` -> `feedInfoAfterUpdates`
  - `round` -> `roundAfterUpdates`
  - `value` -> `valueAfterUpdates`
  - `actualData` -> `historyData`